apps/showcase/src/assets/trainings/sdk/steps/typescript-sdk/exercise/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, inject, signal } from '@angular/core';
+import { type Pet, PetApi } from 'sdk';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  /** Title of the application */
+  public title = 'tutorial-app';
+
+  private readonly petStoreApi = inject(PetApi);
+  readonly #pets = signal<Pet[]>([]);
```

Comment:
```suggestion
  public readonly #pets = signal<Pet[]>([]);
```

Reply:
the `#` prefix means that the variable is private, so there is no need to rewrite `private` before (in fact it causes an error)